src/common/hotbar-store.ts

```diff
@@ -0,0 +1,67 @@
+import { action, comparer, observable, toJS } from "mobx";
+import { BaseStore } from "./base-store";
+import migrations from "../migrations/hotbar-store";
+
+export interface HotbarItem {
+  entity: {
+    uid: string;
+  };
+  params?: {
+    [key: string]: string;
+  }
```

Comment:
I don't think we should care what the param types are.

```suggestion
  params?: Record<string, any>; 
```